src/interfaces/swig/Library.i

```diff
@@ -8,11 +8,8 @@
 #define PROTOCOLS_SGVECTOR(class_name, type_name, format_str, typecode)
 #endif
 
-%shared_ptr(shogun::Cache)
-%shared_ptr(shogun::ListElement)
 %shared_ptr(shogun::Signal)
 %shared_ptr(shogun::Time)
```

Comment:
I think it’s OK to remove Time? I mean all languages have ways to measure process walltime and cpu time

Reply:
Remove

Reply:
gone :)